exporter/exporterhelper/internal/persistent_queue.go

```diff
@@ -87,19 +87,19 @@ var (
 )
 
 // NewPersistentQueue creates a new queue backed by file storage; name and signal must be a unique combination that identifies the queue storage
-func NewPersistentQueue[T any](sizer Sizer[T], capacity int, dataType component.DataType, storageID component.ID,
-	marshaler func(req T) ([]byte, error), unmarshaler func([]byte) (T, error), set exporter.CreateSettings) Queue[T] {
-	_, isRequestSized := sizer.(*RequestSizer[T])
+func NewPersistentQueue[T any](set QueueCreateSettings[T], storageID component.ID, marshaler func(req T) ([]byte,
```

Comment:
Why not having a PersistentQueueCreateSettings? Also, to be honest not sure if we should use `Create` here, we use in the Factory case since that is a `CreateProcessor` function, but in case of the `service` objects I think we don't use.

Reply:
Makes sense. I renamed it to `QueueSettings/PersistentQueueSettings` in https://github.com/open-telemetry/opentelemetry-collector/pull/9197